docs/src/apis/gats.md

```diff
@@ -1,290 +0,0 @@
-# Theories, instances, and expressions
```

Comment:
I know some of this is dated, but it is good content. Is it worth porting the relevant bits to the Gatlab docs? Or do you already have some comparable high-level exposition there?

Reply:
Yes, I agree, this should be moved to the GATlab docs. We have some comparable high-level exposition, but we might be missing some things.